src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -350,14 +350,8 @@ private static bool IsTypeModifierOrTypeKeyword(SyntaxKind kind)
                 case SyntaxKind.SealedKeyword:
                 case SyntaxKind.StaticKeyword:
                 case SyntaxKind.UnsafeKeyword:
-<<<<<<< HEAD
-                case SyntaxKind.OpenBracketToken:
                 case SyntaxKind.ExplicitKeyword:
                 case SyntaxKind.ImplicitKeyword:
-||||||| e9f1cd29802
-                case SyntaxKind.OpenBracketToken:
```

Comment:
It looks like ```SyntaxKind.OpenBracketToken``` got dropped. Is this intentional or am I misinterpreting the merge?

Reply:
It looks like I misinterpreted the merge